dietpi/dietpi-software

```diff
@@ -11615,6 +11655,76 @@ _EOF_
 			G_EXEC chown -R ffsync:ffsync /mnt/dietpi_userdata/firefox-sync
 
 		fi
+		
+                software_id=183 # Bitwarden RS
+                if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+                    Banner_Configuration
+
+                    Create_User -d /mnt/dietpi_userdata/bitwarden_rs bitwarden_rs
+
+                    # Make dir needed to run
+                    G_EXEC mkdir -p /mnt/dietpi_userdata/bitwarden_rs/
+
+                    # Make and edit the template config to have no web vault
+		    if [[ ! -f '/mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env' ]]
+		    then
+			    G_EXEC cp /opt/bitwarden_rs/.env.template /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
+			    G_CONFIG_INJECT 'WEB_VAULT_ENABLED=' 'WEB_VAULT_ENABLED=true' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
+			    G_CONFIG_INJECT 'DATA_FOLDER=' 'DATA_FOLDER=/mnt/dietpi_userdata/bitwarden_rs' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
+                    fi
+		    
+                    # Download and extract the web vault
+                    cd /mnt/dietpi_userdata/bitwarden_rs
+                    local version=$(curl -sf "https://api.github.com/repos/dani-garcia/bw_web_builds/releases/latest" | grep -m1 '^[[:blank:]]*"tag_name":' | cut -d \" -f 4)
+                    G_EXEC wget -c "https://github.com/dani-garcia/bw_web_builds/releases/download/$version/bw_web_$version.tar.gz" -O - | tar -xz
```

Comment:
Since the URL ships a `browser_download_url`, and `Download_Install` automated download and extraction:
```
Download_Install "$(curl -sf 'https://api.github.com/repos/dani-garcia/bw_web_builds/releases/latest' | mawk -F\" '/browser_download_url.*\.tar\.gz\"/{print $4;exit}')" /mnt/dietpi_userdata/bitwarden_rs
```
An it does not really make a difference, but maybe this fits better into the install step?

Reply:
OK, that makes more sense

Reply:
Jep, it does it based on the file type for a set of archive types, zip, 7z, tarballs (xz is still missing 😅) + installs DEB packages.